lib/teselagen/api/client.py

```diff
@@ -361,8 +366,9 @@ def select_laboratory(self, lab_name: Optional[str]=None, lab_id: Optional[int]=
 
     def unselect_laboratory(self) -> None:
         """ Clear the selection of a laboratory and removes it from instance headers."""
-        del self.headers["tg-active-lab-id"]
-        print(f"No Lab is now selected.")
+        if "tg-active-lab-id" in self.headers:
+            del self.headers["tg-active-lab-id"]
+        print(f"Selected Lab: Common")
```

Comment:
We could document what happens implicitly when removing `"tg-active-lab-id"` from the headers, and where it happens.

Reply:
Done